packages/types/src/extensions/pn-any-address-based-types.ts

```diff
@@ -1,17 +1,18 @@
 import * as Extension from '../extension-types';
 
 /** Manager of the extension */
-export interface IAddressBased extends Extension.IExtension {
+export interface IAddressBased<TCreationParameters = ICreationParameters>
+  extends Extension.IExtension {
   createAddPaymentAddressAction: (
     creationParameters: IAddPaymentAddressParameters,
   ) => Extension.IAction;
   createAddRefundAddressAction: (
     creationParameters: IAddRefundAddressParameters,
   ) => Extension.IAction;
   createCreationAction: (
-    creationParameters: ICreationParameters,
-  ) => Extension.IAction<ICreationParameters>;
-  isValidAddress: (address: string) => boolean;
+    creationParameters: TCreationParameters,
+  ) => Extension.IAction<TCreationParameters>;
+  supportedNetworks: string[];
```

Comment:
`isValidAddress()` has always been protected, it was a mistake to declare it in this interface (and not needed now with inheritance)